components/Themes.tsx

```diff
@@ -11,3 +11,11 @@ export const invertAccentColor = (T) => {
   theme.colors.secondaryVariant = brand[400];
   return theme;
 };
+
+export const invertHighlightColor = (T) => {
```

Comment:
should we combine the `invertHighlightColor` and the `invertAccentColor` into the same theme? I feel like since they'd both be dark mode we can just have one theme for that - idk do y'all have any thoughts on that

Reply:
maybe we could have an `invertTheme` or `darkMode` function that calls the two functions? i don't know if that'd be cleaner or messier, but it gives us the option to only apply certain inversions if we want to without having to rewrite stuff

Reply:
yea I think this would be a good idea especially if there's any other dark theme stuff that comes up later on